packages/material-ui-lab/src/MasonryItem/MasonryItem.test.js

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { createClientRender, describeConformanceV5 } from 'test/utils';
+import MasonryItem, { masonryItemClasses as classes } from '@material-ui/lab/MasonryItem';
+import { expect } from 'chai';
+import Masonry from '../Masonry/Masonry';
+
+describe('<MasonryItem />', () => {
+  const render = createClientRender();
+
+  describeConformanceV5(<MasonryItem height={100} />, () => ({
+    classes,
+    inheritComponent: 'div',
+    render,
+    refInstanceof: window.HTMLDivElement,
+    testComponentPropWith: 'div',
+    muiName: 'MuiMasonryItem',
+    skip: ['componentsProp'],
+  }));
+
+  const children = <div data-testid="test-children" />;
+  const testHeight = 100;
+  it('should render with the root class by default', () => {
+    const { getByTestId } = render(
+      <MasonryItem height={testHeight} data-testid="test-root">
+        {children}
+      </MasonryItem>,
+    );
+    expect(getByTestId('test-root')).to.have.class(classes.root);
+  });
```

Comment:
If I am not mistaken, this is being tested in the conformance test suite. Can you check, maybe by breaking it and seeing if the conformance test will fail?